File: 240307/폭탄 해제/bomb-removal.py
class Mission:
  def __init__(self, code, color, sec) :
    self.code = code
    self.color = color
    self.sec = sec

code, color, sec = input().split()
mission1 = Mission(code, color, int(sec))

print(f'code : {mission1.code} \ncolor : {mission1.color}\nsecond : {mission1.sec}')

File: 240424/폭탄 해제/bomb-removal.js
class Bomb {
    constructor(code, color, sec) {
        this.code = code;
        this.color = color;
        this.sec = sec;
    }
}

const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split(' ')
const bomb1 = new Bomb(input[0], input[1], input[2])

console.log(`code : ${bomb1.code} \ncolor : ${bomb1.color} \nsecond : ${bomb1.sec}`)